dataplane/linux/vxlan_mgr_test.go

```diff
@@ -0,0 +1,203 @@
+package intdataplane
+
+import (
+	"net"
+	"time"
+
+	"github.com/projectcalico/felix/rules"
+
+	"github.com/projectcalico/felix/ip"
+
+	"github.com/projectcalico/felix/routetable"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/projectcalico/felix/proto"
+	"github.com/vishvananda/netlink"
+)
+
+type mockVXLANDataplan struct {
+	links []netlink.Link
+}
+
+func (mockVXLAND *mockVXLANDataplan) LinkByName(name string) (netlink.Link, error) {
```

Comment:
nit: generally it's common practice to use shorter names for structures in bound functions like this.

So, it might be more idiomatic to do `(m *mockVXLANDataplane)`